drivers/regulator/regulator_shell.c

```diff
@@ -109,6 +109,98 @@ static int cmd_set_ilim(const struct shell *sh, size_t argc, char **argv)
 	return 0;
 }
 
+static int cmd_set_mode(const struct shell *sh, size_t argc, char **argv)
+{
+	int mode, ret;
+	const struct device *reg_dev;
+
+	reg_dev = device_get_binding(argv[1]);
+	if (!device_is_ready(reg_dev)) {
+		shell_error(sh, "regulator device %s not available", argv[1]);
+		return -ENODEV;
+	}
+
+	mode = atoi(argv[2]);
```

Comment:
This is just nitpicking, but `strto*` functions are preferred over `atoi` IIRC. @stephanosio

Reply:
Nits make code better :-). I've added a commit to this PR to cleanup all usages of `atoi` in this shell module, and switched the new commands to use `strtol` as well

Reply:
Indeed, `strto*` should be preferred over `atoi` because they provide better error handling.